.vscode/launch.json

```diff
@@ -2,16 +2,24 @@
     "version": "0.2.0",
     "configurations": [
         {
-            "type": "gdb",
-            "request": "attach",
-            "gdbpath": "${workspaceFolder}/toolchain/arm-none-eabi/bin/arm-none-eabi-gdb",
-            "name": "Debug",
-            "executable": "${workspaceFolder}/kernel/build/kernel.sym",
-            "target": ":1234",
-            "remote": true,
-            "cwd": "${workspaceRoot}",
-            // "preLaunchTask": "run_debug",
-            "valuesFormatting": "prettyPrinters"
+            "name": "(gdb) Attach",
+            "type": "cppdbg",
+            "request": "launch",
+            "program": "${workspaceFolder}/src/kernel/build/kernel.sym",
+            "cwd": "${workspaceFolder}/src/kernel/",
+            "miDebuggerPath": "${workspaceFolder}/src/toolchain/arm-none-eabi/bin/arm-none-eabi-gdb",
```

Comment:
I have `arm-none-eabi-gdb` installed through my package manager so this configuration doesn't work as it complains the path doesn't exist. Maybe it's possible to add `${workspaceFolder}/src/toolchain/arm-none-eabi/bin/arm-none-eabi-gdb` to the runners `PATH` and then just specify `arm-none-eabi-gdb` here?

Reply:
Good idea, will do!